activiti-cloud-starter-runtime-bundle/src/test/java/org/activiti/cloud/starter/tests/runtime/SignalIT.java

```diff
@@ -75,11 +79,29 @@ public void setUp() throws Exception {
         }
     }
 
+    @Test
+    public void shouldBroadcastSignals() throws Exception {
```

Comment:
@daisuke-yoshimoto the test should validate that both:
1) local behaviour is working as expected
2) an event is published if the Cloud behaviour is present.

Reply:
@Salaboy I changed the test to validate that both.

Reply:
👍 +1